index.html

```diff
@@ -51,6 +51,109 @@ <h1 class="introduction">I'm Ivan<br>Glad to see you</h1>
+            <div class="tags">
+                <img src="./images/html.png" alt="html tag" class="work-tags">
+                <img src="./images/css.png" alt="css tag" class="work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+            </div>
+            <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+        </div>
+        <div class="work">
+            <img src="./images/work4.svg" alt="screenshot of first project" class="work1_pic">
+            <h3 class="work-name">Multi-Post Stories</h3>
+            <div class="work-data">
+                <strong class="company"> Canopy </strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="role">Back-End Dev</strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="year">2015</strong>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <img src="./images/html.png" alt="html tag" class="work-tags">
+                <img src="./images/css.png" alt="css tag" class="work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+            </div>
+            <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+        </div>
+    </section>
+    <section class="about">
+        <h2 class="about-me">About<br>Myself</h2>
+        <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="connect-about">Let's connect</a>
+        <div>
+            <ul class="social-about">
+                <li class="social-icon-about"><a href="#" class="twitter"><img src="./images/twitter.png" alt="twitter icon"></a></li>
+                <li class="social-icon-about"><a href="#" class="linkedin"><img src="./images/linkedin.png" alt="linkedin icon"></a>
+                </li>
+                <li class="social-icon-about"><a href="#" class="medium"><img src="./images/medium.png" alt="medium icon"></a>
+                </li>
+                <li class="social-icon-about"><a href="#" class="github"><img src="./images/github.png" alt="github icon"></a>
+                </li>
+                <li class="social-icon-about"><a href="#" class="angellist"><img src="./images/angellist.png" alt="angellist icon"></a></li>
+            </ul>
+        </div>
+        <a href="#" class="see-project"><img src="./images/resume.svg" alt="Button that links to my resume"></a>
```

Comment:
- [ ] [OPTIONAL] I kindly recommend you to put this `<a>` tag inside a `<div>` or any other container tag, as this will help your code to be cleaner.